src/filesystem/streams/read.js

```diff
@@ -0,0 +1,213 @@
+// NOTE: this is a raw re-implementation of https://github.com/nodejs/node/blob/v10.x/lib/internal/fs/streams.js#L45
```

Comment:
can we add the node license here

Reply:
we are already using NodeJS to run this, so we already depend on it. Do u still think we need it here? (also the referenced file doesn't have a License at the beginning)

Reply:
@jbudz I've added the node license itself in the license of the project just like node does https://github.com/elastic/static-fs/blob/master/LICENSE